html-templates/cbl/dashboards/_template.tpl

```diff
@@ -21,7 +21,7 @@
     <main class="content site" role="main">
         {block "iframe"}
             <iframe
-                src="https://building21.looker.com/embed/dashboards/{$dashboardId}?theme=StudentPLP&allow_login_screen=true"
+                src="https://building21.looker.com/embed/{$dashboardType}/{$dashboardId}?theme={$theme}&allow_login_screen=true"
```

Comment:
```suggestion
                src="https://building21.looker.com/embed/{$dashboardType|default:dashboards}/{$dashboardId}?theme={$theme|default:Staff_StudentPLP}&allow_login_screen=true"
```

@tgaffey FYI, if you did it like this you'd only have to set them per-template when you want to override the default